pyvrp/search/tests/test_Route.py

```diff
@@ -342,18 +344,15 @@ def test_route_tws_access(ok_small):
     the route.
     """
     route = Route(ok_small, idx=0, vehicle_type=0)
-    for client in range(1, ok_small.num_clients + 1):
+    for client in range(ok_small.num_depots, ok_small.dimension):
         route.append(Node(loc=client))
 
     route.update()
 
     for loc in [0, 1, 2, 3, 4, 5]:  # = [depot, *clients, depot]
```

Comment:
This hardcodes the length of the route which the rest of the test doesn't so I'd suggest
```suggestion
    for loc in range(len(route) + 2):  # = [depot, *clients, depot]
```